pyproject.toml

```diff
@@ -176,8 +164,7 @@ mkdocs = "~1.2.3"
 # Integration Tests
 requests = "~2.27.1"
 # Selenium web drivers for live integration testing
-# NOTE: selenium 4.x drops support for python 3.6
-selenium = "~3.141.0"
+selenium = "~4.1.2"
```

Comment:
Not sure if this means we should also update the Selenium container version (https://github.com/nautobot/nautobot/blob/develop/development/docker-compose.yml#L88) as well - probably so.

Reply:
on next branch: its on latest tag
>   https://github.com/nautobot/nautobot/blob/e896524ea57aa887bfd88f0267cfbc060743929b/development/docker-compose.yml#L88

Reply:
> on next branch: its on latest tag
> 
> > https://github.com/nautobot/nautobot/blob/e896524ea57aa887bfd88f0267cfbc060743929b/development/docker-compose.yml#L88

Yeah but this was addressed in `develop`. `next` needs to be brought up to date with what was just merged into `main` yesterday.

Reply:
👍